.github/workflows/build-verification.yml

```
@@ -45,310 +45,323 @@ jobs:
+
+          cat report.md
+
+      - name: Create GitHub check
+        uses: LouisBrunner/checks-action@v1.6.2
+        if: always()
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          name: Build Verification
+          conclusion: ${{ steps.parse-report.outputs.status == 'PASS' && 'success' || 'failure' }}
+          output: |
           {
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 357: could not find expected ':'

(syntax-check)